hermes-common/src/test/groovy/pl/allegro/tech/hermes/common/schema/ReadMetricsTrackingRawSchemaClientTest.groovy

```diff
@@ -0,0 +1,108 @@
+package pl.allegro.tech.hermes.common.schema
+
+import com.codahale.metrics.Timer
+import pl.allegro.tech.hermes.api.RawSchema
+import pl.allegro.tech.hermes.api.TopicName
+import pl.allegro.tech.hermes.common.metric.HermesMetrics
+import pl.allegro.tech.hermes.common.metric.Timers
+import pl.allegro.tech.hermes.schema.RawSchemaClient
+import pl.allegro.tech.hermes.schema.SchemaVersion
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+import static pl.allegro.tech.hermes.common.schema.SchemaRepositoryType.SCHEMA_REGISTRY
+
+class ReadMetricsTrackingRawSchemaClientTest extends Specification {
```

Comment:
I couldn't find a way to unit test such class, since it only decorates specific implementation.
This test only checks if specific methods are wrapped with appropriate timers calls.

I'll add integration test with `GraphiteMock`